.github/workflows/build.yml

```diff
@@ -15,28 +15,28 @@ jobs:
   build: # make sure build/ci work properly
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@85e6279cec87321a52edac9c87bce653a07cf6c2  # v4.2.2
```

Comment:
That is not true?
v4.2.2 has the hash `11bd71901bbe5b1630ceea73d27597364c9af683`
https://github.com/actions/checkout/commit/11bd71901bbe5b1630ceea73d27597364c9af683

Reply:
No,
what the comment at the line's end represents is the most current release version of the used action, while in the workflow the latest commit has is used.